solr/core/src/test/org/apache/solr/core/BlobRepositoryMockingTest.java

```diff
@@ -71,7 +71,8 @@ public void setUp() throws Exception {
     super.setUp();
     blobFetched = false;
     blobKey = "";
-    reset(mocks);
+    reset(mockContainer);
```

Comment:
Can we clean up the assignments at line 51 and 54 since some of those are obsolete now I think?

Reply:
Not getting your point. Please elaborate.

Reply:
With this change, do we still use mocks field? Or is it used elsewhere and only skipped in this one test?

Reply:
we don't have `mocks` field anymore https://github.com/apache/solr/blame/main/solr/core/src/test/org/apache/solr/core/BlobRepositoryMockingTest.java